src/stores/widgetManager.ts

```diff
@@ -870,6 +870,21 @@ export const useWidgetManagerStore = defineStore('widget-manager', () => {
     profile.hash = corrDefault?.hash ?? profile.hash
   })
 
+  const copyWidgetToView = (widget: Widget, viewName: string): boolean => {
+    let cloned = false
+    currentProfile.value.views.some((view) => {
+      if (view.name === viewName) {
+        const newWidget = JSON.parse(JSON.stringify(widget))
+        newWidget.hash = uuid4()
+        view.widgets.unshift(newWidget)
+        cloned = true
+        return true
+      }
+      return false
+    })
+    return cloned
+  }
+
```

Comment:
Instead of returning true or false, as this is an "action" function, it should just "return void", this is, return nothing if everything goes right and throw an error in case something goes wrong.

Returning true or false in "action" functions is an old practice from languages like C that didn't have mecahisms to raise/throw errors.